.github/actions/install-cli/action.yml

```diff
@@ -1,17 +1,17 @@
-name: "Install Hyperlane CLI"
-description: "Install the Hyperlane CLI by packaging from a local build"
+name: 'Install Hyperlane CLI'
```

Comment:
is it the prettier default to be ' instead of "" now?

Reply:
Good question. The default is to use double quotes, but our configuration sets it to single quotes (which, incidentally, matches my personal preference).

I think the reason it’s changing in this PR is that we never applied formatting to some configuration files, including this directory.